mplc/contributivity.py

```diff
@@ -938,6 +937,13 @@ def without_replacment_SMC(self, the_scenario, sv_accuracy=0.01, alpha=0.95):
             end = timer()
             self.computation_time_sec = end - start
 
+    def Flip_label(self):
+        mpl = multi_partner_learning.MplLabelFlip(self.scenario)
+        mpl.fit()
+        self.contributivity_scores = [np.linalg.norm(
+            mpl.theta[-1][i] - np.identity(mpl.theta[-1][i].shape[0])
+        ) for i in range(len(self.scenario.partners_list))]
+
```

Comment:
As the contribution score is supposed to be high when there is no cheating I would rather put:
`  np.sqrt(mpl.theta[-1][i].shape[0]) - np.linalg.norm( mpl.theta[-1][i] - np.identity(mpl.theta[-1][i].shape[0]) )`